AmericanMedicalAssociation/AMA-style-guide
# Bugfix/make better gift

## Ticket(s)

Please do not submit a Pull Request without a relevant ticket! If you are creating a new pattern or feature, create an issue describing the need for this feature Github **first** and then link to it below.

**Github Issue**
NA

**Jira Ticket**

- [EWL-4266: Deploy is broken](https://issues.ama-assn.org/browse/EWL-4266)


## Description

This adds the `patch-package` package to enable patching of npm modules. Using this required updating to a much more recent version of `npm`, which also changes how lock files are managed. We'll now be switching from shrinkwrap to npm's native lock files. I've also added a new requirements section to make clear the version dependencies.

## To Test

- Read the readme and update your local to be consistent with the given requirements. Run `npm --version` to make sure you're up to >=5.2 (I'm using v5.5.1).
- Run `npm install` to update all the things. If anything breaks, run `rm -rf node_modules && npm install`
- Run `gulp serve` and make sure things look good
- Run `gulp drupal-deploy` to make sure it works (this should be ok to test, as this was based on develop pretty close to the time of last release).


## Relevant Screenshots/GIFs

Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks

When this is merged we'll need to notify everyone to update their NPM version ASAP.

## Additional Notes

Anything more to add?
